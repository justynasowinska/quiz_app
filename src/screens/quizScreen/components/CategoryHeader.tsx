import * as React from 'react';
import { Text, StyleSheet } from 'react-native';

import { THEME_COLORS } from '../../../config/colors';

interface PropsType {
    category: string;
}

export const CategoryHeader = (props: PropsType) => {
    return (
        <Text style={styles.header}>
            {props.category}
        </Text>
    );
};

const styles = StyleSheet.create({
    header: {
        fontSize: 22,
        fontWeight: 'bold',
        color: THEME_COLORS.headerTextColor,
        textAlign: 'center'
    }
});