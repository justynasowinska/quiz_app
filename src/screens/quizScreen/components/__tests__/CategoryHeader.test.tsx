import * as React from 'react';
import { shallow } from 'enzyme';

import { CategoryHeader } from '../CategoryHeader';

describe('Quiz Screen CategoryHeader Component', () => {
    it('matches to the snapshot', () => {
        const component = shallow(<CategoryHeader category="History" />);

        expect(component).toMatchSnapshot();
    });
});
