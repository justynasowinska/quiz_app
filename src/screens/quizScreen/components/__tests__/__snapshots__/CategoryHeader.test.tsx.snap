// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz Screen CategoryHeader Component matches to the snapshot 1`] = `
<Text
  style={
    Object {
      "color": "#0C9198",
      "fontSize": 36,
      "fontWeight": "bold",
      "textAlign": "center",
    }
  }
>
  History
</Text>
`;
